sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultKeyStoreTest.java

```diff
@@ -155,4 +159,27 @@ public void testEngineStore() {
         KeyVaultKeyStore keystore = new KeyVaultKeyStore();
         keystore.engineStore(null, null);
     }
+
+    @Test
+    public void testRefreshEngineGetCertificate() throws Exception {
+        System.setProperty("azure.keyvault.jca.refresh-certificates-when-have-un-trust-certificate", "true");
+        KeyVaultJcaProvider provider = new KeyVaultJcaProvider();
+        Security.addProvider(provider);
+        KeyStore ks = PropertyConvertorUtils.getKeyVaultKeyStore();
+        Certificate certificate = ks.getCertificate("myalias");
+        ks.deleteEntry("myalias");
+        Thread.sleep(10);
```

Comment:
Can we delete `Thread.sleep(10);`?

Reply:
Because the time interval between `ks.getCertificate("myalias");` and `ks.getCertificateAlias(certificate)` is too short, the value of `lastRefreshTime` set in `ks.getCertificate("myalias");` is the same as the `lastForceRefreshTime` time set in `ks.getCertificateAlias(certificate)`, which will cause a test error